### PR TITLE
Possiblity to choose between docker and podman from the command line

### PR DESCRIPTION
**Pull request by j-porsche**
### Description

This PR adds the possibility to choose between docker and podman to run the docker.sh script from the command line. From now on you are able to add a flag after the script (-p) to run the script with podman. @weizhouapache For backwardwards compatibility, docker is used by default.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by j-porsche** [comment]
@svenvogel @utchoang

**Comment by j-porsche** [comment]
This PR was already made but with some mistakes. From now on I will manage this PR. The old one was: Add selection for podman/docker #5141

**Review by DennisKonrad — APPROVED** [review]
LGTM

**Comment by j-porsche** [comment]
@weizhouapache May You take a look at it?

**Comment by j-porsche** [comment]
@weizhouapache Could You take a look at it?

File: ui/docker.sh [hunk 1]
```diff
@@ -29,5 +29,19 @@ DATE="$(date --iso-8601=seconds)"
...
-
+USE_PODMAN=0
+USE_DOCKER=0
+OPTIONS='pd'
+while getopts $OPTIONS OPTION
+do
+        case "$OPTION" in
+                p)USE_PODMAN=1;;
+                d)USE_DOCKER=1;;
+                *)echo "Please choose between -p for podman or -d for docker.";;
```
Comment (weizhouapache) [inline ui/docker.sh:40]: @j-porsche `exit` at  line 40 ?

Comment (j-porsche) [reply ui/docker.sh:40]: @weizhouapache Yeah. Good point. I'll add this

**Comment by j-porsche** [comment]
@weizhouapache I have updated the file. Could You please look at it again. Thank You :)

**Review by weizhouapache — APPROVED** [review]
code LGTM.
did not test with podman or docker.

**Pull request merged**